DataFormats/CTPPSAlignment/src/RPAlignmentCorrectionsData.cc

```diff
@@ -58,15 +60,18 @@ RPAlignmentCorrectionData RPAlignmentCorrectionsData::GetFullSensorCorrection(un
   bool useRPErrors) const
 {
   RPAlignmentCorrectionData c;
-  mapType::const_iterator it = rps.find(id / 10);
-  if (it != rps.end())
-    c = it->second;
-  it = sensors.find(id);
-  if (it != sensors.end())
-    c.add(it->second, useRPErrors);
 
-  //printf("> full correction %u | ", id);
-  //c.print();
+  // try to get alignment correction of the full RP
+  auto rpIt = rps.find(CTPPSDetId(id).getRPId());
```

Comment:
`id` represents a sensor id. CTPPS sensors are enclosed in so called Roman Pots (RPs). `CTPPSDetId(id).getRPId()` returns the id of the RP hosting the sensor described by `id`.

The old version `id / 10` was a left over from the old TOTEM SW, where we used decimal ids.